src/jwk.rs

```diff
@@ -299,6 +300,30 @@ impl JWK {
         Ok(JWK::from(Params::EC(ec_params)))
     }
 
+    #[cfg(feature = "openssl")]
+    pub fn generate_p384() -> Result<JWK, Error> {
+        let group = openssl::ec::EcGroup::from_curve_name(openssl::nid::Nid::SECP384R1)?;
+        let keypair = openssl::ec::EcKey::generate(&group)?;
+        let form = openssl::ec::PointConversionForm::UNCOMPRESSED;
+        let mut ctx = openssl::bn::BigNumContext::new()?;
+        let sk_bytes = keypair.private_key().to_vec();
+        if sk_bytes.len() != 48 {
```

Comment:
Have you looked at https://docs.rs/openssl/0.10.40/openssl/bn/struct.BigNumRef.html#method.num_bytes? Maybe I misunderstoof the method though. Otherwise I think it needs explicit zerioising, and there's also a `clear` method.

Reply:
`sk_bytes` is returned in the JWK. Where is zeroize needed?

Reply:
Oh right, I guess it isn't then